### PR TITLE
[FEATURE] Header

### PR DESCRIPTION
**Pull request by ramonxm**
## Description

Criação da Header da aplicação
\

## Screenshots / Videos

image

## Developer Checks

- [X] PR title & commits adhere to [Conventional Commit Specification](https://www.conventionalcommits.org/en/v1.0.0/)
- [X] PR is targeting correct branch, is up-to-date, & no merge conflicts
- [X] Tested on browser device
- [X] Tested on Mobile device

### CODE REVIEW

**Comment by jonataspinto** [comment]
PQP 31 files

**Comment by jonataspinto** [comment]
depois do carnaval eu termino 😆

**Review by jonataspinto — COMMENTED** [review]
| Device | URL |
|--|--|
| desktop | http://estartandodevs-site.s3-website-sa-east-1.amazonaws.com/ |

     
| Device | URL |
|--|--|
| mobile | http://estartandodevs-site.s3-website-sa-east-1.amazonaws.com/ |

Not what you expected? Are your scores flaky? [Run Lighthouse on Foo](https://www.foo.software/lighthouse)
If scores continue to be inconsistent consider [running all audits on Foo](https://github.com/foo-software/lighthouse-check-action#usage-foos-automated-lighthouse-check-api)

**Comment by jonataspinto** [comment]
<details>
    <summary>Lighthouse result for https://www.estartandodevs.com.br/</summary>
    

  | Metric | Score |  |
  | --- | --- | --- |
  | Performance | 0.55 | ✅ |
| First Contentful Paint | 3.7 s |  |
| Time to Interactive | 6.5 s |  |
| Speed Index | 4.8 s |  |
| Total Blocking Time | 620 ms |  |
| Largest Contentful Paint | 4.3 s |  |
| Cumulative Layout Shift | 0 |  |

    
</details>
<details>
    <summary>Lighthouse result for http://estartandodevs-site.s3-website-sa-east-1.amazonaws.com/</summary>
    

  | Metric | Score |  |
  | --- | --- | --- |
  | Performance | 0.97 | ✅ |
| First Contentful Paint | 1.4 s |  |
| Time to Interactive | 4.8 s |  |
| Speed Index | 2.3 s |  |
| Total Blocking Time | 120 ms |  |
| Largest Contentful Paint | 1.4 s |  |
| Cumulative Layout Shift | 0.004 |  |

    
</details>

File: src/components/Header/styles.ts [hunk 1]
```diff
@@ -0,0 +1,66 @@
+import styled from 'styled-components';
+
+export const ContainerHeader = styled.header`
+  display: flex;
+  align-items: center;
+  max-width: 100%;
```
Comment (jonataspinto) [inline src/components/Header/styles.ts:6]: width somente não?

File: src/components/Header/styles.ts [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+
+  @media (max-width: 425px) {
+    padding: 0 16px;
+  }
+`;
+
+export const SectionHeader = styled.section`
+  display: flex;
+  gap: 48px;
+  align-items: flex-start;
```
Comment (jonataspinto) [inline src/components/Header/styles.ts:28]: display flex já é start nos eixos

File: src/components/LinkWrapper/styles.ts [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+
+export const ContainerLink = styled.div<ColorProps>`
+  all: unset;
+  cursor: pointer;
+
+  display: flex;
+  align-items: center;
+  justify-content: center;
+
+  font-size: ${convertPxToRem(17)};
```
Comment (jonataspinto) [inline src/components/LinkWrapper/styles.ts:16]: achei que iria remover.

**Review by jonataspinto — CHANGES_REQUESTED** [review]

File: src/components/Header/styles.ts (hunk 2 above)
Comment (ramonxm) [reply src/components/Header/styles.ts:28]: sim, esqueci de tirar isso ai

File: src/components/LinkWrapper/styles.ts (hunk 3 above)
Comment (ramonxm) [reply src/components/LinkWrapper/styles.ts:16]: deixar só no global?

**Comment by jonataspinto** [comment]
<details>
    <summary>Lighthouse result for https://www.estartandodevs.com.br/</summary>
    

  | Metric | Score |  |
  | --- | --- | --- |
  | Performance | 0.62 | ✅ |
| First Contentful Paint | 3.1 s |  |
| Time to Interactive | 4.8 s |  |
| Speed Index | 4.7 s |  |
| Total Blocking Time | 470 ms |  |
| Largest Contentful Paint | 4.9 s |  |
| Cumulative Layout Shift | 0 |  |

    
</details>
<details>
    <summary>Lighthouse result for http://estartandodevs-site.s3-website-sa-east-1.amazonaws.com/</summary>
    

  | Metric | Score |  |
  | --- | --- | --- |
  | Performance | 0.94 | ✅ |
| First Contentful Paint | 1.7 s |  |
| Time to Interactive | 5.2 s |  |
| Speed Index | 2.7 s |  |
| Total Blocking Time | 80 ms |  |
| Largest Contentful Paint | 2.2 s |  |
| Cumulative Layout Shift | 0.011 |  |

    
</details>

**Review by jonataspinto — COMMENTED** [review]
| Device | URL |
|--|--|
| desktop | http://estartandodevs-site.s3-website-sa-east-1.amazonaws.com/ |

     
| Device | URL |
|--|--|
| mobile | http://estartandodevs-site.s3-website-sa-east-1.amazonaws.com/ |

Not what you expected? Are your scores flaky? [Run Lighthouse on Foo](https://www.foo.software/lighthouse)
If scores continue to be inconsistent consider [running all audits on Foo](https://github.com/foo-software/lighthouse-check-action#usage-foos-automated-lighthouse-check-api)

File: src/components/LinkWrapper/styles.ts (hunk 3 above)
Comment (jonataspinto) [reply src/components/LinkWrapper/styles.ts:16]: falo do convertPxToRem

**Review by jonataspinto — APPROVED** [review]
LGTM

**Comment by jonataspinto** [comment]
>      
> | Device | URL |
> |--|--|
> | desktop | http://estartandodevs-site.s3-website-sa-east-1.amazonaws.com/ |
> 
>      
> | Device | URL |
> |--|--|
> | mobile | http://estartandodevs-site.s3-website-sa-east-1.amazonaws.com/ |
> 
> Not what you expected? Are your scores flaky? [Run Lighthouse on Foo](https://www.foo.software/lighthouse)
> If scores continue to be inconsistent consider [running all audits on Foo](https://github.com/foo-software/lighthouse-check-action#usage-foos-automated-lighthouse-check-api)
> 
> 

@ramonxm pwa subiu um cadinho 🤔

**Comment by jonataspinto** [comment]
Merregeeeeeee

**Pull request merged**